src/js/components/tablero/Filter.jsx

```diff
@@ -63,76 +75,101 @@ class Filter extends Component {
     this.setState({ filterCategorySelected: !this.state.filterCategorySelected });
   }
 
+  loadStoredFilters = (pageFilters) => {
+    if (this.props.pageFilters.length > 0) {
```

Comment:
Use destructuring at the begining.
`const { pageFilters, activeConfig } = this.props;`

Reply:
Done